src/base/PhpBlock.php

```diff
@@ -67,4 +67,32 @@ public function renderAdmin()
         $this->injectorSetup();
         return $this->admin();
     }
+    
+    public function onRegister()
+    {
+        if ($this->getIsCacheEnabled()) {
+            $phpBlockView = $this->getView();
+    
+            $blockId = $this->getEnvOption('id');
+    
+            $cacheKeyAssets = ['blockassets', $blockId];
+            $cacheKeyAssetBundles = ['blockassetbundles', $blockId];
+            
+            $assets = Yii::$app->cache->getOrSet($cacheKeyAssets, [$phpBlockView, 'getBlockAssets'], $this->getCacheExpirationTime());
```

Comment:
We should implement the Cachable trait and use those functions instead of Yii::$app->cache (https://luya.io/api/luya-traits-CacheableTrait)

Reply:
There we get some conflicts with the $cacheEnabled and $cacheExpiration property of `CacheableTrait` and `InternalBaseBlock`. So I implemented `CacheableTrait` into InternalBaseBlock to resolve the conflicts.